tiledb/sm/query/sparse_global_order_reader.cc

```diff
@@ -170,6 +170,9 @@ Status SparseGlobalOrderReader::dowork() {
   std::vector<tuple<>> buffers;
   for (auto& buffer : buffers_) {
     names.emplace_back(buffer.first);
+    if (buffer.first == constants::timestamps) {
+      load_timestamps_ = true;
```

Comment:
Sparse global order reader always loads timestamps as it needs to do deduplication.